ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -263,9 +263,10 @@ private IEnumerable<NotificationSite> CreateNotificationSitesFromModel(Notificat
             }
         }
 
-        public ContentResult OnGetValidateClinicalDetailsDate(string key, string day, string month, string year)
+        public async Task<ContentResult> OnGetValidateClinicalDetailsDate(string key, string day, string month, string year)
         {
-            return ValidationService.GetDateValidationResult<ClinicalDetails>(key, day, month, year);
+            var notification = await NotificationRepository.GetNotificationAsync(NotificationId);
```

Comment:
If all we need from these methods (and the other instances of dynamic validation where we're pulling in the notification to determine whether legacy or not - is it worth exposing a `IsNotificationLegacyAsync` type method?

Reply:
Good point! I'll give that a go now